addons/website_event/static/src/scss/event_templates_list.scss

```diff
@@ -115,4 +117,7 @@
             color: #fff;
         }
     }
+    .o_half_screen_height {
+        min-height: 200px !important;
```

Comment:
A bit strange, looks like an override of editor / website class?
It's not based on "vh" height anymore.

If there is a very good reason to hardcode 200px, could be nice to explain it with a comment.